Getting started guide

GBKS opened the pull request:
Do not merge, this is a work-in-progress branch PR.

ConorOkus commented on guide/getting-started/1-getting-started.md:
```diff
@@ -0,0 +1,23 @@
+
+## By the people, for the people
+
+One of the unique aspects of Bitcoin is that it is an open-source project and no single group, company or government owns or controls it. That means all of us can freely participate and contribute as much or as little as we want. All contributions are equally valid, and there are no gatekeepers. This decentralization is an incredibly important property for a global currency as it provides a level ...
```
I'd be hesitant to call it a `global currency` as I feel it implies it will be used as currency is today and there is no guarantee of that given the prevailing digital gold narrative. I'd be okay with `global transfer of value` maybe?

moneyball replied:
Different people have different aspirations for bitcoin. My own opinion and one I advocate for with this community is to aspire to become a global currency. I think we should design with that in mind. This is not inconsistent with bitcoin today as speculative investment to become a SoV or digital gold - in fact IMO it is a necessary step to become a global currency. I literally do not see how it could be a global currency if it did not go through a monetization phase of $0 to something on the order of trillions of dollars of market cap (based on the size of M0 or the size of gold).

As long as we are not overpromising that _today's_ bitcoin is a usable global currency then I have no issue with us using that language and working toward it.

GBKS replied:
:) thanks for the input. I've rewritten this page already multiple times and expect to do so again. It's a tricky one to get right. One option for this particular phrasing of global currency is to describe bitcoin as something new with a unique combination of properties that allows it be used in different ways, and that these uses are expanding as it gets adopted more widely. Which I think is also why there is a much bigger cryptocurrency/altcoin ecosystem, people are trying to figure out what else is possible with this new thing.

GBKS replied:
I ended up removing this copy when restructuring the guide hierarchy.

ConorOkus commented on guide/getting-started/1-getting-started.md:
```diff
@@ -0,0 +1,23 @@
+
+This guide is an example of this. It was created and is maintained by a community of designers and design enthusiasts that want to help improve the overall level of design in the Bitcoin ecosystem. It is continuously improved and you are welcome to help, whether it’s providing feedback or adding more content.
+
+While there are many businesses offering Bitcoin services, this guide is dedicated primarily to non-custodial user experiences. Just like you are fully in control of the cash you carry in your wallet, non-custodial Bitcoin experiences mean that users fully own and control their Bitcoin. Because this puts more responsibility on users, these experiences are much harder to design and implement. Yet, ...
```
Awesome, very clear 👍

ConorOkus commented on guide/getting-started/11-common-user-flows.md:
```diff
@@ -0,0 +1,38 @@
+Onboarding may be purely informational based on the content, but many times it also includes an initial setup that helps personalize the software towards the users specific needs and context.
+
+## Creating a new wallet
+I is important here to distinguish between wallet software, and the wallet that represents the users "account". This particular experience may be almost invisible or highly elaborate. While some mobile wallet applications choose to automatically create and secure a wallet during first launch, other wallets provide extensive configuration wizards.
```
```suggestion
It is important here to distinguish between wallet software, and the wallet that represents the users "account". This particular experience may be almost invisible or highly elaborate. While some mobile wallet applications choose to automatically create and secure a wallet during first launch, other wallets provide extensive configuration wizards.
```

ConorOkus commented on guide/getting-started/11-common-user-flows.md:
```diff
@@ -0,0 +1,38 @@
+## Wallet maintenance
+
+## Resolving a problem
+This can be a tricky experience to address. For one, non-custodial cryptocurrency management by nature places a lot of responsibility on the user. This also puts much of the burden of solving problems on users. The second aspect is that open-source software typically relies on online documentation and forums for "customer service". Ideally, software helps prevent errors as much as possible throug ...
```
I think applications should really consider "customer support" as part of the safety net for resolving users problems. Although an agent may not be able to remedy things that happen on the network it can be helpful for someone to explain what has happened when an error or something unexpected occurs

GBKS replied:
For sure. It's a pretty unthankful and often neglected job to write proper error messages and help sections, but it's super helpful for those tricky situations when something goes wrong.

ConorOkus commented on guide/getting-started/12-technology-primer.md:
```diff
@@ -0,0 +1,30 @@
+
+It can take many years to understand bitcoin in every detail, but luckily you don’t need to know everything to design great products. Let’s take a look at the most important aspects of the technology behind bitcoin, with an eye towards the basics needed to design bitcoin products.
+
+The most fundamental read is the Bitcoin White Paper. Published on October 31, 2008 by the mysterious Satoshi Nakamoto, it layous the fundamental system design in only 9 short pages.
```
```suggestion
The most fundamental read is the Bitcoin White Paper. Published on October 31, 2008 by the mysterious Satoshi Nakamoto, it lays out the fundamental system design in only 9 short pages.
```

ConorOkus commented on guide/getting-started/12-technology-primer.md:
```diff
@@ -0,0 +1,30 @@
+
+This mechanism of generating a piece of information from another one, where one can be shared and the other one is kept private is known as public-key cryptography. While there are many other cryptographic techniques used in Bitcoin, this is one that you should understand well.
+
+So you may ask how Bitcoin are created? This involves a process we know as mining. Miners are computers that help process transactions. For this effort, which requires huge amounts of computing power, they are rewarded with freshly minted Bitcoin. At the moment, 6.25 new Bitcoin are created around every 10 minutes, which is an annual inflation rate of 1.86%. The creation of Bitcoin is tightly con ...
```
I would say one of the properties of good money is scarcity but not the only thing that gives it value. e.g durability, portability, fungible, divisible etc

GBKS replied:
That leads us back to the other point you made about bitcoin being (or not being) a global currency, and how that is defined. Maybe we do need a section on that, or link to a good resource that explains it ([example](https://www.investopedia.com/ask/answers/100314/why-do-bitcoins-have-value.asp)). We might end up having to refer to these properties in a lot of other sections, too.

ConorOkus commented on guide/getting-started/9-personal-finance.md:
```diff
@@ -0,0 +1,198 @@
+   layout = "float-left"
+%}
+
+Rent, utilities, subscriptions, car payments, grocery bills and similar expenses need to be paid regularly on a weekly or monthly basis. Compared to discretionary day-to-day spending, these payments are higher in value, less frequent and play a more important role in our lives. We can got without an extra coffee in the afternoon, but not paying rent or water bills is typically not an option. 
```
```suggestion
Rent, utilities, subscriptions, car payments, grocery bills and similar expenses need to be paid regularly on a weekly or monthly basis. Compared to discretionary day-to-day spending, these payments are higher in value, less frequent and play a more important role in our lives. We can go without an extra coffee in the afternoon, but not paying rent or water bills is typically not an option. 
```

ConorOkus commented on guide/getting-started/9-personal-finance.md:
```diff
@@ -0,0 +1,198 @@
+
+While grocery bills are still one-off payments and we may visit different stores every week, many payments in this category are tied to specific service providers we have relationships with. Financial institutions also provide different mechanisms for recurring payments where businesses can regularly deduct money from accounts without customers having to confirm each one. Bitcoin transactions hav ...
+
+The amounts involved in these transactions are higher and therefore require better security. While I can may cash with me and hand it to anyone I desire directly, monthly expenses are typically done indirectly via transfers between the banks of both parties. A manual payment might require me to log into my banks website via username and password, along with a second security mechanism like [two-f ...
```
```suggestion
The amounts involved in these transactions are higher and therefore require better security. While I can carry cash with me and hand it to anyone I desire directly, monthly expenses are typically done indirectly via transfers between the banks of both parties. A manual payment might require me to log into my banks website via username and password, along with a second security mechanism like [two-factor authentication](https://en.wikipedia.org/wiki/Help:Two-factor_authentication){:target="_blank"}.
```

ConorOkus commented on guide/getting-started/9-personal-finance.md:
```diff
@@ -0,0 +1,198 @@
+   layout = "float-left"
+%}
+
+A sinking fund is a way to systematically set money aside for a planned large expense, like a new car or a down payment for a new home. The amount may be similar to emergency budgets, but the sinking fund is created for a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal or transfer after which the fund is no longer needed. This use case does not  ...
```
```suggestion
A sinking fund is a way to systematically set money aside for a planned large expense, like a new car or a down payment for a new home. The amount may be similar to emergency budgets, but the sinking fund is created for a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal or transfer after which the fund is no longer needed. This use case does not necessarily require a dedicated account. Based on the amount, it could be covered with a dedicated account in either the wallet for monthly budgeting or savings.
```

GBKS replied:
Hm, I don't see a change... did you remove a space or something along those lines?

ConorOkus replied:
Oh weird, yeh I think it was just removing some extra space

ConorOkus commented on guide/getting-started/9-personal-finance.md:
```diff
@@ -0,0 +1,198 @@
+   layout = "float-left"
+%}
+
+Investing is about putting saved money to work in the hope that it increases in value, from retirement accounts to stock investing and real estate ownership. This necessarily requires the involvement of third parties, which goes beyond the current  focus of this guide.
```
```suggestion
Investing is about putting saved money to work in the hope that it increases in value, from retirement accounts to stock investing and real estate ownership. This necessarily requires the involvement of third parties, which goes beyond the current focus of this guide.
```

ConorOkus commented on guide/getting-started/14-hardware.md:
```diff
@@ -0,0 +1,76 @@
+   height = 800
+%}
+
+Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on and can easily catch up with network activity, Lightning nodes need to be online at all times in order to maintain payment channels with other nodes. Smartphones are typically also online 24/7 but are not suited well for this task due to large amounts of data tr ...
```
```suggestion
Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on and can easily catch up with network activity, Lightning nodes need to be online at all times in order to maintain payment channels with other nodes. Smartphones are typically also online 24/7 but are not suited well for this task due to large amounts of data transfer and storage capacity requirements.
```

Rspigler commented on guide/getting-started/12-technology-primer.md:
```diff
@@ -0,0 +1,30 @@
+
+This mechanism of generating a piece of information from another one, where one can be shared and the other one is kept private is known as public-key cryptography. While there are many other cryptographic techniques used in Bitcoin, this is one that you should understand well.
+
+So you may ask how Bitcoin are created? This involves a process we know as mining. Miners are computers that help process transactions. For this effort, which requires huge amounts of computing power, they are rewarded with freshly minted Bitcoin. At the moment, 6.25 new Bitcoin are created around every 10 minutes, which is an annual inflation rate of 1.86%. The creation of Bitcoin is tightly con ...
```
> At the moment, 6.25 new Bitcoin are created around every 10 minutes, which is an annual inflation rate of 1.86%.

Perhaps after the above sentence add something like:  "Approximately every 4 years, this amount is reduced by 50%, resulting in a supply capped at 21 million"

GBKS replied:
I ended up removing this part. I am considering a page on bitcoin economics on which this information would be a natural fit, but right now it seems secondary to other content that is more focused on design.

ConorOkus commented on guide/getting-started/11-common-user-flows.md:
```diff
@@ -1,38 +1,223 @@
+   layout = "float-left"
+%}
+
+Mobile app stores do a good job at providing previews of what using an app will be like. Through copy, videos, images and reviews, users can make informed decisions about the product the are evaluating. Open-source software is typically downloaded via a website or from Github and each project decides what information to present. 
```
```suggestion
Mobile app stores do a good job at providing previews of what using an app will be like. Through copy, videos, images and reviews, users can make informed decisions about the product they are evaluating. Open-source software is typically downloaded via a website or from Github and each project decides what information to present. 
```

ConorOkus commented on guide/getting-started/14-hardware.md:
```diff
@@ -20,7 +20,7 @@ Generally, any hardware with an internet connection can interact with the bitcoi
 ## Recovery phrase storage
 
-Simple, cheap and robust storage solutions for recovery phrases come in modest forms. [Seedplate](https://bitcoinseedbackup.com/), for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com), in comparison, is the size of a credit card and comes pre-loaded with a recovery phrase (hidden behind a s ...
+Simple, cheap and robust storage solutions for recovery phrases come in modest forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, in comparison, is the size of a credit card and comes pre-loaded with  ...
```
I'm vehemently against writing down recovery phrases but have used https://shieldfolio.com/ in the past and it's pretty good I think. Really durable and the ghost pen provides low visibility where the seeds are only visible under UV light.

GBKS replied:
I think recovery phrases are actually one of the most powerful things about bitcoin because they allow anyone with a pen and paper (or good memory) to have custody their bitcoin. You don't even need a computer. Not ideal if you have access to better tech, but still an important aspect.

Shieldfolio looks cool, but a bit expensive for a notebook and an invisible ink pen. But definitely way cooler than if you just bought generic ones off of Amazon.

pavlenex commented on guide/getting-started/12-technology-primer.md:
```diff
@@ -0,0 +1,208 @@
+
+It can take many years to understand bitcoin in every detail, but luckily you don’t need to know everything to design great products. Let’s take a look at the most important ideas and technolgies behind Bitcoin, with an eye towards the basics you may need to design bitcoin products.
+
+The most fundamental read is the Bitcoin White Paper. Published on October 31, 2008 by the mysterious Satoshi Nakamoto, it lays out the fundamental system design in only 9 short pages.
```
> the Bitcoin White Paper

perhaps a link to the whitepaper?

pavlenex commented on guide/getting-started/12-technology-primer.md:
```diff
@@ -0,0 +1,208 @@
+   layout = "float-right"
+%}
+
+From the mempool, transactions are bundled into blocks. One a transaction is in a block that the network has accepted, the transaction is considered "confirmed". Blocks are created roughly every 10 minutes.
```
Typo `When`?

pavlenex commented on guide/getting-started/13-software.md:
```diff
@@ -0,0 +1,166 @@
+   layout = "float-right-desktop"
+%}
+
+A node connects to and participates in the bitcoin network to download and exchange transaction data. Some wallets have built-in nodes and some node software also supports wallet functionality. The minimum implementation of a node however is that of syncing data and allowing other software a user has to access it.
```
> The minimum implementation of a node however is that of syncing data and allowing other software a user has to access it.
	
Could be me, but I don't quite understand this sentence.  _is that of syncing data_ is confusing to me.

pavlenex commented on guide/getting-started/14-hardware.md:
```diff
@@ -0,0 +1,82 @@
+
+## Hardware wallets
+
+You may already be familar with key fobs from your bank or work. They are small devices with specific built-in authentication that is used to control and secure access to a computer system. Key fobs are not connected to the internet or any other network, so they cannot easily be hacked without physical access.
```
While I like and understand this example, people in third-world countries like myself may not have an idea of what a key fob is. Don't have a personal preference, just want to point out that if our goal is worldwide audience, we probably should use something a bit more generic, though I can't think of anything atm, so just consider this as a suggestion for the future.

johnsBeharry replied:
👋 third-world country guy here too.

Never heard of key fobs, had to google 😅

pavlenex commented on guide/getting-started/14-hardware.md:
```diff
@@ -0,0 +1,82 @@
+
+## Nodes
+
+Thousands of computers participate in the Bitcoin network. They are called nodes and continuously exchange, verify and compare transaction data. Running your own node, instead of relying on a third-party node, has the benefit that transactions are getting validated by your own personal device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products have  ...
```
> Several DIY and consumer hardware products have been created to make it easy to run your own node on a dedicated device.
	
IMO this give an impression that you need a consumer-grade device to run a node. You can run a node on your computer just by running adequate software.

pavlenex reviewed (commented):
Had some time to go through this and made some minor suggestions, great work on how you broke down complex stuff into easy to digest parts, yet relatable to one another.

johnsBeharry commented on guide/getting-started/12-technology-primer.md:
```diff
@@ -0,0 +1,281 @@
+   layout = "float-right-desktop"
+%}
+
+This function is performed by miners, which are computers that group new transactions from the mempool into blocks and try to get those blocks accepted by the Bitcoin network. They are called miners because the creation of new blocks is rewarded by freshly minted bitcoin. This is the only way bitcoin are created, so there is heavy competition around mining.
```
An important function of the miners is that they verify that an output can be spent by checking those in the transaction against the UTXO set.

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+Licenses help ensure that bad actors cannot abuse this open system by taking the published work and pretend it's their own, patent it, redistribute it under a commercial license, or otherwise profit from it. A variety of licenses have been created by the open-source movement to accomodate different types of projects and situations.
+
+This guide is published under three different licenses ([MIT license](LICENSE-MIT), [Apache License 2.0](LICENSE-APACHE), and [Creative Commons Attribution 4.0 International License](LICENSE-CC-BY)). Anyone who wants to remix it, or parts of it, may choose which one to publish their work under. You can read the full license text [here](LICENSE).
```
Remember to add the links in for these 👍

GBKS replied:
Good call.

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+Open design tries to balance the independence of individual designers and the collective power of collaboration. The goal is to enable and promote design projects, which are motivated by personal conviction and passion of designers for the greater benefit of a global society.
+
+In practical terms, open design can manifest itself in a different ways. You might:
```
```suggestion
In practical terms, open design can manifest itself in different ways. You might:
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+Whichever it may be, it is a voluntary, public contribution to efforts that you think are worthwhile. From these individual efforts, a network of people and projects emerges that is only loosely bound through personal relationships, communities, or the work itself. This is different from common corporate structures and requires individuals to think and act more independently and be comfortable wi ...
+
+Open-source development has existed and matured for decades, while open design is not as widely known or practiced. Starting to design for bitcoin products can therefore sometimes be disorienting, as many designers need to adjust not just how they work and collaborate, but also the relationship to their work. No longer is the design an artefact only controlled by just the designer or a small grou ...
```
```suggestion
Open-source development has existed and matured for decades, while open design is not as widely known or practiced. Starting to design for bitcoin products can therefore sometimes be disorienting, as many designers need to adjust not just how they work and collaborate, but also the relationship to their work. No longer is the design an artefact only controlled by just the designer or a small group of people inside an organization. The design instead becomes part of a public conversation where anyone is invited to help shape it, and group consensus influences the way forward. In open-source software, this public review and debate often lead to better products and services, and this effect ideally carries over to open design.
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+Open-source development has existed and matured for decades, while open design is not as widely known or practiced. Starting to design for bitcoin products can therefore sometimes be disorienting, as many designers need to adjust not just how they work and collaborate, but also the relationship to their work. No longer is the design an artefact only controlled by just the designer or a small grou ...
+
+The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translaters, project managers, and others working together.
```
```suggestion
The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translators, project managers, and others working together.
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+##  Open design and Bitcoin
+
+Bitcoin has had the spirit of openness deeply embedded from its humble beginnings. The source code is public, participation in the network is public and unrestricted, and transfering value via the Bitcoin protocoal is permissionless as well. Thousands of projects have copied ideas, techniques and code from Bitcoin to experiment and explore different directions. So it provides a fantastic foundati ...
```
```suggestion
Bitcoin has had the spirit of openness deeply embedded from its humble beginnings. The source code is public, participation in the network is public and unrestricted, and transferring value via the Bitcoin protocol is permissionless as well. Thousands of projects have copied ideas, techniques and code from Bitcoin to experiment and explore different directions. So it provides a fantastic foundation for designers to also contribute in this open and participatory manner.
```

GBKS replied:
Too bad it's wrong. Protocoal sounds like a cool thing :)

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+- make it easy for others to use and build on our ideas and work
+- are inclusive and help others
+- are open-minded and flexible 
+- work multidisciplinary, because good results require different perspectives and skillsets
```
```suggestion
- work multidisciplinary, because good results require different perspectives and skill sets
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+## Getting started
+
+If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
```
```suggestion
If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting point is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
+
+- Join the [Bitcoin Design Community](http://bitcoindesigners.org) on Slack and see what others are discussing. A good way to get started is by asking questions and joining conversations that interest you. Our [Community Calls](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) and [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen ...
+- Download a Bitcoin wallet and experiment with small amounts. You can alse use Testnet, which allows you use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.
```
```suggestion
- Download a Bitcoin wallet and experiment with small amounts. You can also use Testnet, which allows you to use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+- Download a Bitcoin wallet and experiment with small amounts. You can alse use Testnet, which allows you use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.
+- Redesign a Bitcoin application of your choice. Even as a brief exercise, a redesign is a great way to develop deeper understanding of how something works and why certain decisions were made.
+- Read, watch and listen. There is a lot of great material out there to learn more about Bitcoin. Design-focused content is unfortunately not as common but expanding into related materials (like finance) might help.
+- Review your tools and workflows, and the work you produce, and consider how you can make it more public and accesssible for others.
```
```suggestion
- Review your tools and workflows, and the work you produce, and consider how you can make it more public and accessible for others.
```

ConorOkus commented on guide/getting-started/open-design.md:
```diff
@@ -0,0 +1,112 @@
+
+---
+
+There is enourmous space for creativity. To find a starting point, take a look at the [visual-language of bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }}).
```
```suggestion
There is enormous space for creativity. To find a starting point, take a look at the [visual-language of bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }}).
```

GBKS commented:
> Yeh I think that could be good, as there is quite a range now

@ConorOkus I created [an issue for a page about hardware wallets](https://github.com/BitcoinDesign/Guide/issues/59) to address this (at some point in the future).

Rspigler commented on guide/getting-started/hardware.md:
```diff
@@ -0,0 +1,96 @@
+
+Hardware wallets for bitcoin play a similar role but with several differences. Similar to key fobs, they securely store an authentication mechanism and isolate it from other devices and networks. In the case of bitcoin, these are the seed, associated keys and all functions that require direct interaction with them.
+
+Because hardware wallets are so optimized for this specific uses case, they have very small screens and limited input options. Most interactions with them happen via desktop software that is also provided by the manufacturer. So users perform most wallet interactions on their desktops, and the application sends only specific requests to the hardware wallet to verify or sign data. This provides a  ...
```
Because hardware wallets are so optimized for this specific use case... (not uses case)

GBKS replied:
Fixed. Thank you.

Rspigler commented on guide/getting-started/software.md:
```diff
@@ -0,0 +1,175 @@
+
+# Software overview
+
+Finance is complex and there are many use cases, from giving weekly allowance to a child and selling goods in a store to managing multi-billion dollar retirement funds and hectic day trading. So we end up with many different applications solutions for the various uses cases. Bitcoin being a global currency adds another layer of complexity as the software may need to be adapted to different langua ...
```
I think it would be:
"So we end up with many different applications..."
or
"So we end up with many different application solutions..."

either way, it is "...for the various use cases" at the end.

ConorOkus replied:
You can make suggested changes which are handy for typos and grammar - https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

Rspigler replied:
Hm, that's how I thought I did it....

GBKS replied:
Added the changes. Thank you.

Rspigler reviewed (changes requested):
Grammar comments

Rspigler commented on guide/getting-started/hardware.md:
```diff
@@ -0,0 +1,96 @@
+
+## Nodes
+
+Thousands of computers participate in the Bitcoin network. They are called nodes and continuously exchange, verify and compare transaction data. Running your own node, instead of relying on a third-party node, has the benefit that transactions are getting validated by your own personal device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products have  ...
```
Thousands of computers participate in the Bitcoin network. They are called nodes and continuously exchange, verify and compare transaction and block data. Running your own node, instead of relying on a third-party node, has the benefit that transactions are getting validated by your own personal device. Nodes also improve privacy and help secure the network. Several DIY and consumer hardware products have been created to make it easy to run your own node on a dedicated device.

Rspigler replied:
@ConorOkus yep, still didn't do it right, sorry 😳

ConorOkus replied:
There is no right way to do this so no worries, was just a suggestion 👍

GBKS replied:
Added the changes. Thank you.

danielnordh commented on guide/getting-started/user-research.md:
```diff
@@ -0,0 +1,191 @@
+
+This has resulted in diverse and lively online commmunities that are deeply invested in bitcoin. Designers and researchers have vast amounts of public conversation at their hands that allow them to better understand user needs and issues. These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.
+
+Some of the following links have been around for a long day and provide a huge archive and history of information and conversation around bitcoin:
```
Suggest changing to '...for a long time...' instead of '...for a long day...'

GBKS replied:
Fixed, thank you.

GBKS commented:
Alright, the section is in good shape, although there's still a lot to refine overall. But I'd still like to merge it into the main branch for now. This will make it easier to tackle individual sections individually, and also allow for easier integration with the other sections that are being worked on (others have said that this would be helpful).

I ticked off the following pre-merge items last week:
- Checked for dead links
- Removed unused files
- Optimized images
- Added editor’s notes for each page
- Added image source attribution
- Added image alt texts

Could you please ACK, NACK, review, discuss, praise, tweak, improve, complain, and anything else you'd like? Bring it all.

danielnordh commented:
ACK!
Agree that merging this will make it easier for other chapters, plus it's a great step forward overall.
Good job.

ConorOkus commented:
ACK, good job @GBKS

pavlenex commented on guide/getting-started/getting-started.md:
```diff
@@ -2,9 +2,102 @@
 
-This chapter is in development at [PR #20](https://github.com/BitcoinDesign/Guide/pull/20){:target="_blank"}. Check the [preview](https://gbks.github.io/Guide/getting-started/introduction/){:target="_blank"}.  
\ No newline at end of file
+Welcome to bitcoin design, you're in for an adventure. This chapter is all about the big concepts that are helpful to be familar with when creating bitcoin products. You'll find a brief summary of each concept onn this page, click through if you would like to dig deeper into a specific topic. Note that this is not meant as a comprehensive overview of all topics bitcoin, but only the ones we think ...
```
```suggestion
Welcome to bitcoin design, you're in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating bitcoin products. You'll find a brief summary of each concept on this page, click through if you would like to dig deeper into a specific topic. Note that this is not meant as a comprehensive overview of all topics bitcoin, but only the ones we think are important to know as a designer working in this space.
```

pavlenex commented on guide/getting-started/getting-started.md:
```diff
@@ -2,9 +2,102 @@
+
+**[Why bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})**
+
+The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transfering value that could bring similar change as previous waves of innovation.
```
```suggestion
The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
```

pavlenex reviewed (commented):
tACK (tested locally) with just a few typos I noticed along the way. 

This is impressive amount of work Christoph. There are things that can be ironed out at a later point, but I agree with overall feeling that this is needed as a foundation to kick things off.

johnsBeharry commented:
tACK. I've been using the the tech primer, open source, why bitcoin is unique and why design for bitcoin pages in the mentorship workshops and its already proving really useful so I think getting it released gives us a good foundation to build from. Great work!

johnsBeharry reviewed (approved):
tACK. I've been using the the tech primer, open source, why bitcoin is unique and why design for bitcoin pages in the mentorship workshops and its already proving really useful so I think getting it released gives us a good foundation to build from. Great work!
